README.md

```diff
@@ -23,6 +23,21 @@ A core responsibility of the Sidecar is facilitating the calculations of [reward
 * MacOS
 * Linux (Ubuntu/Debian)
 
+# Diagram
```

Comment:
Doesnt belong in this particular context. Eventually we'll have more detailed docs that show the architecture of the sidecar.